annotation/tests/test_jobs.py

```diff
@@ -468,6 +473,9 @@ def test_validate_job_extensive_coverage_error(
 def test_update_job_files(files: Tuple[File]):
     mock_session = MagicMock()
     patch_data = {"files": {1, 2}, "datasets": {"dataset1"}}
+    for i in range(2):
+        files[i].distributed_annotating_pages = None
+        files[i].distributed_validating_pages = None
```

Comment:
Could you elaborate on this change?

Reply:
I added these two fields for the other test and because the function update_job_files creates files without distributed_annotating_pages and distributed_validating_pages the assertion fails. So I turn these two fields into None so I can still use the fixture and the assertion will be successful

Reply:
It is not really clear, could you adjust your fixture or create a new one? You can make a fixture to accept arguments